concepts/operator-reconciliation/README.md

```diff
@@ -0,0 +1,168 @@
+# Operator-based Reconciler
+
+## Motivation
+
+Reconciler is a framework to install, update, and repair Kyma components in managed Kyma Runtimes. The reconciliation runs in the loop to ensure that all components are up and running and properly configured.
+While we currently have a custom reconciliation logic in place, we could make use of the Kubernetes Control Loop and reconciliation and an Operator-based approach to maintain our reconciliation through Kubernetes natively.
+
+We are currently having these issues in our current reconciliation:
+
+- Scheduling / Job-based
+  - We have inbuilt limits on how fast a reconciliation can work by having bad worst case scheduling scenarios (e.g. reconciliation updates only get picked up every 45 seconds through our bookkeeper)
+  - Reconciliation must be externally managed from outside the Cluster as there is no internal state management inside a shoot, this requires either the CLI or a control plane cluster with a Mothership
```

Comment:
Using the Kyma CLI you can also use it on your own cluster as well, also private gardener shoots?

Reply:
That would have been my expectation, I still think you will have a provision and deploy command. So if you already have your private gardener shoot, you only need to deploy with the kubeconfig